x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -16268,6 +16268,7 @@
     "xpack.securitySolution.autocomplete.selectField": "请首先选择字段......",
     "xpack.securitySolution.beatFields.errorSearchDescription": "获取 Beat 字段时发生错误",
     "xpack.securitySolution.beatFields.failSearchDescription": "无法对 Beat 字段执行搜索",
+    "xpack.securitySolution.callouts.dismissButton": "关闭",
```

Comment:
We've got a translation team that goes in and fills out the new or updated i18n fields, you just have to delete any unused translations in these files